dotnet/src/SemanticKernel/KernelBuilder.cs

```diff
@@ -70,13 +81,65 @@ public IKernel Build()
     /// </summary>
     /// <param name="log">Logger to add.</param>
     /// <returns>Updated kernel builder including the logger.</returns>
+    [Obsolete("This method is deprecated and will be removed in one of the next SK SDK versions. Use AddLogging instead.")]
```

Comment:
I know this is gonna be annoying... but can you revert this Obsolete and undo the may files that only include this change? This rename should just come in another PR.

I'd keep the new Add* methods, including AddLogging (so for a while, we have both). Then just internally change the implementation of "WithLogger" to call "AddLogger".
Then, in another PR, we can submit this renaming with a bunch of similar breaking changes ("AddAzureOpenAI", etc)